src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds

```diff
@@ -37,5 +37,24 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroName)' == 'rhel'" Include="rhel/Microsoft.NETCore.Runtime.CoreCLR-symbols.pkgproj">
```

Comment:
Should it be “.symbols” (lowercase) or “.Symbols” (uppercase)? The project files determine the name of the nuget packages.

mikem

From: Davis Goodin [mailto:notifications@github.com]
Sent: Wednesday, April 6, 2016 7:58 AM
To: dotnet/coreclr <coreclr@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>
Subject: Re: [dotnet/coreclr] Add symbol packages for coreclr runtime (#4102)


In src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds<https://github.com/dotnet/coreclr/pull/4102#discussion_r58720171>:

> @@ -37,5 +37,24 @@

>      </Project>

>    </ItemGroup>

>

> +  <ItemGroup Condition="'$(__BuildType)' == 'Release'">

> +    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroName)' == 'rhel'" Include="rhel/Microsoft.NETCore.Runtime.CoreCLR-symbols.pkgproj">

For the nupkg specifically the convention is .symbols.nupkg. At this level .Symbols seems like a decent way to indicate that in a project name.

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/4102/files/5af23158445f3a7067b09c14ec16e9c668ca73aa#r58720171>